components/gitpod-protocol/src/util/stringify.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * Copyright (c) 2024 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+export namespace BigIntToJson {
+    export const BIGINT_KEY = "bigint:";
+    export function replacer(key: string, value: any) {
+        if (typeof value === "bigint") {
+            return `${BIGINT_KEY}${value.toString()}`;
```

Comment:
nit: string can be built by users easier than [object](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt#use_within_json).

Reply:
I know. I just don't like monkey-patching. :slightly_smiling_face: